py/ai/h2o/sparkling/ml/params/H2OGBMParams.py

```diff
@@ -19,18 +19,24 @@
 from pyspark.ml.param import *
 
 from ai.h2o.sparkling.ml.params.H2OSharedTreeParams import H2OSharedTreeParams
+from ai.h2o.sparkling.ml.params.H2OTypeConverters import H2OTypeConverters
 
 
 class H2OGBMParams(H2OSharedTreeParams):
 
     ##
     # Param definitions
     ##
-    learnRate = Param(Params._dummy(), "learnRate", "Learning rate (from 0.0 to 1.0)")
-    learnRateAnnealing = Param(Params._dummy(), "learnRateAnnealing", "Scale the learning rate by this factor after each tree (e.g., 0.99 or 0.999)")
-    colSampleRate = Param(Params._dummy(), "colSampleRate", "Column sample rate (from 0.0 to 1.0)")
-    maxAbsLeafnodePred = Param(Params._dummy(), "maxAbsLeafnodePred", "Maximum absolute value of a leaf node prediction")
-    predNoiseBandwidth = Param(Params._dummy(), "predNoiseBandwidth", "Bandwidth (sigma) of Gaussian multiplicative noise ~N(1,sigma) for tree node predictions")
+    learnRate = Param(Params._dummy(), "learnRate", "Learning rate (from 0.0 to 1.0)",
+                      H2OTypeConverters.toFloat())
```

Comment:
WDYT about checking also ranges of float parameters?

Reply:
Perfect idea -> created jira https://0xdata.atlassian.net/browse/SW-1507